helix-term/src/commands/typed.rs

```diff
@@ -1508,7 +1508,7 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
         TypableCommand {
             name: "set-option",
             aliases: &["set"],
-            doc: "Set a config option at runtime.",
+            doc: "Set a config option at runtime. For example to disable smart case search, use `:set search.smart-case false`.",
```

Comment:
Could add a `\n` in between there to format it as two lines?

Reply:
Ah, can you revert that and use `\n`? I'm fine with it being a single line in the HTML table, I just wanted it to display as two rows in the help popup in command mode

Reply:
Happy to, but it breaks the table rather than being one line--I'll add a screenshot shortly to show the issue.

Reply:
<img width="1382" alt="2022-05-02 at 14 30 47 - Marked 2 - CleanShot@2x" src="https://user-images.githubusercontent.com/70898/166304547-d704d1c2-4ea9-4260-82fe-eacd9d8fb9ea.png">
This is what it looks like with the `\n` in the docs.

This new help message is 18 words, 104 characters. The messages for `debug-remote`, `primary-clipboard-yank-join`, `clipboard-yank-join`, and `write!` are all longer, fwiw.

Reply:
But it does look better in the pop-up as two lines:
<img width="890" alt="2022-05-02 at 14 35 50 - Alacritty - CleanShot@2x" src="https://user-images.githubusercontent.com/70898/166305228-a33255a9-2962-4f14-82eb-d64b99223585.png">
vs
<img width="884" alt="2022-05-02 at 14 38 18 - Alacritty - CleanShot@2x" src="https://user-images.githubusercontent.com/70898/166305535-d9861213-24ac-430f-9d12-5c1246975028.png">

Reply:
Maybe this is something we can fix in the `cargo xtask docgen` that generates the docs? We could have it turn newlines into `<br>`s.

Reply:
Pushing a small patch to do this as suggested. Generates correct docs and correct in-app help:
<img width="1141" alt="2022-05-03 at 13 41 34 - Marked 2 - CleanShot@2x" src="https://user-images.githubusercontent.com/70898/166508963-3bf82ab5-88b2-46de-a18d-687e3b97bd4c.png">
<img width="903" alt="2022-05-03 at 13 42 26 - Alacritty - CleanShot@2x" src="https://user-images.githubusercontent.com/70898/166509033-6c034827-e9c3-4a86-8705-91c059d52ca7.png">

Reply:
@archseer how does this look?

Reply:
Looks great! 🎉